src/main/java/ladder/view/InputView.java

```diff
@@ -0,0 +1,23 @@
+package ladder.view;
+
+import java.util.Scanner;
+
+public class InputView {
+    private final static String PARTICIPANT_QUESTION = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
+    private final static String LADDER_COUNT_QUESTION = "최대 사다리 높이는 몇 개인가요?";
+
+    private final static Scanner scanner = new Scanner(System.in);
```

Comment:
상수로 선언하셨으니, 관례에 따라 이름을 대문자로 작성해보면 어떨까요?
```suggestion
    private final static Scanner SCANNER = new Scanner(System.in);
```